R/convenience-functions.R

```diff
@@ -209,3 +209,68 @@ log_shift <- function(x,
   }
   log(x + offset, base = base)
 }
+
+
+
+
+#' @title Set unit of a single forecast manually
+#'
+#' @description Helper function to set the unit of a single forecast manually.
```

Comment:
Link here to the idea that otherwise will happen automatically and touch on why users might not want this to be the case (and hence why they might use this function)